README.md

```diff
@@ -97,27 +97,43 @@ The options that we support for calculation are as follows:
 To run the globally installed CLI (see above), use the global `fqm-execution command`
 
 ```bash
-Usage: fqm-execution [options]
+Usage: fqm-execution [command] [options]
+
+Commands:
+ detailed (default)
+ raw
+ reports
+ gaps
+ dataRequirements
+ queryInfo
 
 Options:
-  -d, --debug                                 enable debug output (default: false)
-  -o, --output-type <type>                    type of output, "raw", "detailed", "reports", "gaps", "dataRequirements", "queryInfo" (default: "detailed")
-  -r, --report-type <report-type>             type of report, "individual", "summary", "subject-list"
+  --debug                                     enable debug output (default: false)
+  --report-type <report-type>                 type of report, "individual", "summary", "subject-list"
   -m, --measure-bundle <measure-bundle>       path to measure bundle
-  -p, --patient-bundles <patient-bundles...>  paths to patient bundles. Required unless output type is dataRequirements
-  -f, --as-patient-source                     Load bundles by creating cql-exec-fhir PatientSource to pass into library calls
+  -p, --patient-bundles <patient-bundles...>  paths to patient bundles. Required unless output type is dataRequirements. Note: cannot be used with --patient-ids.
+  --patient-ids <ids...>                      (with --fhir-server-url) A list of patient ids an AsyncPatientSource will use to query a fhir server for patient data. Note: cannot be used with --patient-bundles.
+  --as-patient-source                         Load bundles by creating cql-exec-fhir PatientSource to pass into library calls
+  --fhir-server-url <url>                     (with --as-patient-source) Loads bundles into an AsyncPatientSource which queries the passed in fhir server url for patient data
```

Comment:
--fhir-server-url is actually not required with --as-patient-source. Running with just --as-patient-source compiles a normal patient source (as opposed to async), and runs the calculation on that patient source. I do like the idea of adding the note to --fhir-server-url to say it can only be run with --as-patient-source though.